aptos-move/aptos-vm/src/block_executor/mod.rs

```diff
@@ -102,11 +107,12 @@ impl BlockExecutorTransactionOutput for AptosTransactionOutput {
                 (
                     group_key.clone(),
                     group_write.metadata_op().clone(),
-                    // TODO: propagate layouts.
                     group_write
                         .inner_ops()
                         .iter()
-                        .map(|(tag, (op, _maybe_layout))| (tag.clone(), op.clone()))
+                        .map(|(tag, (op, maybe_layout))| {
```

Comment:
this map doesn't change anything anymore, just clones, we should be able to do iter().cloned().collect()

Reply:
Done.